torch/testing/_internal/common_distributed.py

```diff
@@ -356,6 +367,13 @@ def requires_nccl():
     )
 
 
+def requires_xccl():
```

Comment:
If `requires_xccl` is not used in this PR, I suggest removing it and adding it back if necessary.